src/app/service/data-service.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DataService {
+  public data = [{
+    first: [
+      {
+        list: 'first',
+        id: 1,
+        title: 'First list title item',
+        description: 'Example description for first list title item',
+        completed: false,
+      },
+    ],
+    second: [
+      {
+        list: 'second',
+        id: 2,
+        title: 'Second list title item',
+        description: 'Example description for second list title item',
+        completed: false,
+      },
+    ],
+    third: [
+      {
+        list: 'third',
+        id: 3,
+        title: 'Third list title item',
+        description: 'Example description for third list title item',
+        completed: false,
+      },
+    ],
+  }];
+
+  loadAll() {
+    return this.data;
+  }
+  // Add an item in a specific list
+  addItem(list, item) {
+    if (list in this.data[0]) {
+      this.data[0][list].push(item);
```

Comment:
`data[0]` seems wrong, why data is an array but you always use the first element?
Make it an array of lists or an object of lists.